### DISCUSSION TITLE
大佬们，为什么你们这个项目里可以在浏览器端使用@babel/traverse

### DISCUSSION THREAD

**Discussion by maoyaru123** [NetEase/tango#56] [Q&A] [answered]
如题

**Comment by wwsun** [comment, marked as answer]
webpack5 可以参考：https://webpack.js.org/migrate/5/#clean-up-configuration

node polyfill 的构建问题参考：https://stackoverflow.com/questions/64557638/how-to-polyfill-node-core-modules-in-webpack-5
